recipes/libnl/all/test_package/CMakeLists.txt

```diff
@@ -0,0 +1,10 @@
+cmake_minimum_required(VERSION 3.5)
+project(netlink_example)
+
+find_package(Threads REQUIRED)
```

Comment:
If thread is required, you must declare it in `self.cpp_info.system_libs = ["pthread"]`

Reply:
Ok, I will change that.

Reply:
thanks!